fuzzers/030-iob18/Makefile

```diff
@@ -23,6 +23,8 @@ build/segbits_riob18.db: build/segbits_riob18.rdb process_rdb.py bits.dbf
 	sed '/IOB18.IOB_Y1.LVDS/d' -i build/segbits_riob18.rdb
 	python3 process_rdb.py build/segbits_riob18.rdb > build/segbits_riob18_processed.rdb
 	${XRAY_DBFIXUP} --db-root build --zero-db bits.dbf --groups tag_groups.txt --seg-fn-in build/segbits_riob18_processed.rdb --seg-fn-out $@
+	echo 'RIOB18.IOB_Y0.SSTL12_SSTL135_SSTL15.IN !38_126 39_127' >> $@
```

Comment:
I can't fight the feeling that this looks like a hack. I know that this fuzzer is kintex specific, but have never had to do things like this before. I would have to think if there is a better solution for this. For the time being we can open an issue for this and add a comment linking to it.

Reply:
I don't like it much either, unfortunately, but the python script I wrote shows that the bit 39_127 disappears as soon we have a differential input on the Y0 pads, which wipes out the candidates. Unfortunately selectively either setting the tag to 0 or removing the tag in this case caused alterations in the database which did not work on hardware. I am a bit out of ideas of how else I could deal with this, but agree with your proposal to open an issue.

Reply:
I just pushed a fix.